src/Modules/removeTask.js

```diff
@@ -0,0 +1,13 @@
+import { updatingstorage } from './task.js';
+
+const bookremoval = (id, arr) => {
+  const filtered = arr.filter((elem) => elem.index !== id);
+
+  for (let i = 0; i < filtered.length; i += 1) {
+    filtered[i].index = `${i}`;
+  }
```

Comment:
The same with this for loop as the previous suggestion you could try to use a foreach method